.github/workflows/test.yml

```
@@ -54,7 +54,8 @@ jobs:
         terraform:
           - '0.12.30'
           - '0.13.6'
```

Ready for latest versions 👍 

```suggestion
          - '0.12.31'
          - '0.13.7'
```